web/template/attorney/phone_number.gohtml

```diff
@@ -0,0 +1,20 @@
+{{ template "page" . }}
+
+{{ define "pageTitle" }}{{ tr .App "whatIsYourPhoneNumber" }}{{ end }}
+
+{{ define "main" }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form novalidate method="post">
+        <h1 class="govuk-heading-xl">{{ tr .App "whatIsYourPhoneNumber" }}</h1>
+
+        <p class="govuk-body">{{ tr .App "youCanChooseToTellUsYourPhoneNumber" }}</p>
+
+        {{ template "input" (input . "phone" "enterYourUKPhoneNumberOptional" .Form.Phone "hint" "enterYourUKMobileOrLandlineHint" "classes" "govuk-input--width-20 govuk-input--extra-letter-spacing govuk-!-margin-bottom-6" "labelClasses" "govuk-!-font-weight-bold" "type" "tel" "autocomplete" "tel") }}
```

Comment:
`"labelClasses" "govuk-label--s"` instead

Reply:
I also think we should not be adding margin between the control and buttons, we are very inconsistent and it isn't done on the GDS examples (that I've seen)